### PR TITLE
Added documentation on how to configure rocket fuel for local dev

### PR DESCRIPTION
**Pull request by jepp3**
added some documentation for the config.yml file. This is so that new developers can understand the configuration and get the application running locally.

### CODE REVIEW

File: impl/config.example.yml [hunk 1]
```diff
@@ -1,28 +1,57 @@
+# Template Configuration file for local development. You should make a copy of this file
+# and place it in the same folder. The copy of this file should be named config.yml. config.yml is
+# ignored by git and you can change that file without accidentally committing any changes to the
+# configuration ( see git ignore file in the root of the project ).
+
+# Your database connection configuration used when database
+# migrations are executed.
+# The configuration should be identical for the normal database
+# configuration. You should only provide connection information
+# to a a Postgres database, because its the only one supported.
```
Comment (flowertwig) [inline impl/config.example.yml]: ' :)

File: impl/config.example.yml [hunk 2]
```diff
@@ -1,28 +1,57 @@
...
 database:
   user: postgres
   schema: public
   password: mysecretpassword
   url: jdbc:postgresql://localhost:15432/rocket-fuel
 
+# Contains configuration for the application token. The application token
+# is used as session in rocket fuel. The secret is used to verify the integrity
+# of the token. A random string is highly recommended. Even if the value in this
+# file will work. The domain is used  when the application cookie is created.
```
Comment (flowertwig) [inline impl/config.example.yml]: double space

File: impl/config.example.yml (hunk 1 above)
Comment (jepp3) [reply impl/config.example.yml]: fixed :)

File: impl/config.example.yml (hunk 2 above)
Comment (jepp3) [reply impl/config.example.yml]: fixed :)

**Comment by jepp3** [comment]
@kindanice do you merge this change? only documentation

**Pull request merged**